internal/connector/internal/services/connector_cluster.go

```diff
@@ -700,12 +700,20 @@ func (k *connectorClusterService) ReconcileEmptyDeletingClusters(_ context.Conte
 	var errs []*errors.ServiceError
 	if err := k.connectionFactory.New().Transaction(func(dbConn *gorm.DB) error {
 
+		// get client_ids for service accounts
+		var clientIds []string
+		if err := dbConn.Where("id IN ?", clusterIds).
+			Model(&dbapi.ConnectorCluster{}).Select("client_id").
+			Find(&clientIds).Error; err != nil {
+			return services.HandleDeleteError("Connector cluster", "id", clusterIds, err)
+		}
```

Comment:
Shouldn't we check this `err`?

Reply:
What do you mean by _check_ the error? It's a DB search for IDs, if it fails it will be because of some low level DB error, which will be reported in logs as reconciliation errors.

Reply:
I mean that the return of `dbConn.Where` is assigned to `err`, but then the value of `err` is ignored.
I believe we should have an 
```go
if err != nil {
    do something
}
```

Reply:
Sorry, my bad. I didn't see that it is checked.

Ignore this.